sklearn/decomposition/kernel_pca.py

```diff
@@ -79,14 +80,14 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         If int, random_state is the seed used by the random number generator;
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
-        by `np.random`. Used when ``eigen_solver`` == 'arpack'.
+        by ``np.random``. Used when ``eigen_solver`` == 'arpack'.
```

Comment:
Please avoid unrelated changes (I think #13542 will fix this anyway)

When I mentioned double backticks I was only referring to the code you changed ;)

Reply:
:)